### PR TITLE
bump Azure.ResourceManager.Avs to 2022-05-01 API

### PR DESCRIPTION
**Pull request by cataggar**
This bumps the Azure.ResourceManager.Avs to use the 2022-05-01 API that was recently merged in https://github.com/Azure/azure-rest-api-specs/pull/20873 .

### CODE REVIEW

File: sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs [hunk 1]
```diff
@@ -39,8 +39,8 @@ public static partial class AvsExtensions
...
-        public static Azure.Response<Azure.ResourceManager.Avs.Models.AvsSubscriptionTrialAvailabilityResult> CheckAvsTrialAvailability(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.Core.AzureLocation location, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public static System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.Avs.Models.AvsSubscriptionTrialAvailabilityResult>> CheckAvsTrialAvailabilityAsync(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.Core.AzureLocation location, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public static Azure.Response<Azure.ResourceManager.Avs.Models.AvsSubscriptionTrialAvailabilityResult> CheckAvsTrialAvailability(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.Core.AzureLocation location, Azure.ResourceManager.Avs.Models.AvsSku sku = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```
Comment (cataggar) [inline sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs:42]: > ##[error]D:\a\_work\1\.nuget\packages\microsoft.dotnet.apicompat\5.0.0-beta.20467.1\build\Microsoft.DotNet.ApiCompat.targets(82,5): Error : MembersMustExist : Member 'public Azure.Response<Azure.ResourceManager.Avs.Models.AvsSubscriptionTrialAvailabilityResult> Azure.ResourceManager.Avs.AvsExtensions.CheckAvsTrialAvailability(Azure.ResourceManager.Resources.SubscriptionResource, Azure.Core.AzureLocation, System.Threading.CancellationToken)' does not exist in the implementation but it does exist in the contract.

Is this a bug with `microsoft.dotnet.apicompat\5.0.0-beta.20467.1` ? It doesn't seem to handle the `AvsSku sku = null` well. It is in this contract and the implementations.

**Review by cataggar — COMMENTED** [review]
~I think the apicompat checker may have a bug.~

File: sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs (hunk 1 above)
Comment (cataggar) [reply sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs:42]: Nevermind. I read https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#api-compatibility-verification and then made ace36b185e5bb3ed284d549b9e7aff784e304e5c to make it compatible with 1.0.0.

**Comment by cataggar** [comment]
Hi @Yao725, how much more work is it to get the version bumped and published to NuGet?
https://www.nuget.org/packages/Azure.ResourceManager.Avs

**Review by Yao725 — CHANGES_REQUESTED** [review]
Please also update the changelog so we can let the customer know what happened in this upgrade.

**Comment by cataggar** [comment]
@Yao725, the requested changes have been made.

**Review by Yao725 — APPROVED** [review]
LGTM

**Comment by Yao725** [comment]
Hi @cataggar, as this PR is merged, we are going to release the new version 1.1.0 for avs tomorrow, is it ok with you?

**Comment by cataggar** [comment]
@Yao725 Yes. Please do release it.

**Pull request merged**